### ISSUE TITLE
Lightbox render using source tag in picture or video

### ISSUE THREAD

**Issue by lukadriel7** [uikit/uikit#4215]
### Summary

The current lightbox implementation uses the href attribute of links to show images or video.
We are therefore obligated to use a format supported across browser and lose the benefit of more modern format (WebP, Jpeg2000 for images). I use webp on my website with jpeg fallback for safari. Unfortunately, when using lightbox, I have to serve jpeg to all users. An option to choose images format and their fallback would be a great addition for uikit.

### Priority

Importance: 4,
I work on a website which delivers mainly images and targeted towards regions of the world where people have slow and limited mobile data. A few hundreds of kilobyte thus make a great difference.

**Comment by aarongerig** [comment]
Not true, in my case I serve either a webp if the browser supports it or a jpeg if not. I detect that by server headers in PHP. This even massively reduces HTML size served to the client. Maybe worth to come up with a similar solution yourself? ;)

**Comment by lukadriel7** [comment]
Using server side code, sure it is possible. But what about static sites ?

**Comment by picov** [comment]
There is the need to add support for a data-type="picture" for a Lightbox item that inject the whole A content letting the browser to do its job with the selection and load/display of the best supported format.
```
<a href="" data-type="picture">
<picture>
	<source srcset="img1_large.avif 1920w, img1_med.avif 1284w, imgi_small.avif 828w" sizes="100vw" type="image/avif">
	<source srcset="img1_large.webp 1920w, img1_med.webp 1284w, imgi_small.webp 828w" sizes="100vw" type="image/webp">
	<img src="img1_large.jpg" width="1920" height="1080" srcset="img1_large.jpg 1920w, img1_med.jpg 1284w, img1_small.jpg 828w" sizes="100vw">
</picture>
</a>
```
